tox.ini

```diff
@@ -6,7 +6,7 @@ passenv = PGPASSWORD
 commands = py.test {posargs}
 deps =
     requests==2.7.0
-    psycopg2==2.6.0
+    psycopg2==2.7.3.2
```

Comment:
This fixes a problem with the postgres version available on travis. See https://github.com/psycopg/psycopg2/issues/594